test/steps/api/1_given.js

```diff
@@ -14,7 +14,43 @@
  *
  */
 import getAPIClient from '../../../src/utils/api';
+import { getFirstQuotedString } from '../utils';
 
 export function aLiskAPIInstance() {
 	this.test.ctx.liskAPIInstance = getAPIClient();
 }
+
+export function getForgingStatusResolvesWithPublicKey() {
```

Comment:
With the current approach of checking the forging status, the description should also cover the forging value, otherwise the specifications don't make sense.